### DISCUSSION TITLE
Delete/omit individual type files (that were merged)?

### DISCUSSION THREAD

**Discussion by fgblomqvist** [Swatinem/rollup-plugin-dts#173] [Q&A]
This plugin works great so far, but I noticed in the output dir that all the individual type files remain. E.g. I get a merged `index.d.ts` but still have all the `Button.d.ts`, `Header.d.ts` files. Is it possible to tell this plugin to "clean up" as well?

**Comment by wood3n** [comment]
Is there any solutions for this problem? Thanks!

**Reply by DrJKL** [reply]
I'd chain a `rollup-plugin-delete` and specify a subdirectory for declarations.
e.g.
```ts
del({ hook: 'buildEnd', targets: './dist/types' })
```
